client/src/components/Order/OrderForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+
+import Stepper from './Stepper';
+import Stage1 from './Stage1';
+import Stage2 from './Stage2';
+import Button from '@/components/base/Button';
+
+import useInput from '@/hooks/customHooks/useInput';
+
+import { normalRadius } from '@/static/style/common';
+
+const stage1InitialForm = {
+  orderName: '',
+  phoneNumber: '',
+  email: '',
+};
+
+const stage2InitialForm = {
+  recName: '',
+  recPlace: '',
+  recPhoneNumber: '',
+};
+
+const OrderForm = () => {
+  const { form, onChange } = useInput({
+    ...stage1InitialForm,
+    ...stage2InitialForm,
+  });
+
+  const { orderName, phoneNumber, email, recName, recPlace, recPhoneNumber } = form;
+
+  const [stage, setStage] = useState(1);
+
+  const handleClickNext = () => {
+    setStage((prev) => prev + 1);
+  };
+
+  const handleClickPrev = () => {
+    setStage((prev) => prev - 1);
+  };
+
+  const forms = () => {
+    if (stage === 1) {
+      return <Stage1 onChange={onChange} form={{ orderName, email, phoneNumber }} />;
+    } else if (stage === 2) {
+      return <Stage2 onChange={onChange} form={{ recName, recPlace, recPhoneNumber }} />;
+    }
```

Comment:
stage가 0부터 였다면 삼항연산자 사용가능할거같습니다~
```suggestion
    stage  ? <Stage2 onChange={onChange} form={{ recName, recPlace, recPhoneNumber }} /> : 
    <Stage1 onChange={onChange} form={{ orderName, email, phoneNumber }} />;
    
```

Reply:
아 혹시 단계가 더 늘어 날 수도 있어 저렇게 했는데 삼항연산자도 괜찮을 거 같습니다.

Reply:
아하 더 있으면 if가 낫겠네요 👍